samples/openthread/coprocessor/README.rst

```diff
@@ -117,15 +118,24 @@ Building and running
 
 .. _ot_coprocessor_testing:
 
+HCI support
+===========
+
+Currently, HCI is only supported using the nRF USB interface.
+The device will show two virtual UART ports.
+Usually the first port will be associated with the HCI interface, and the second one with the Thread co-processor.
+
 Testing
 =======
 
 After building the sample and programming it to your development kit, complete the following steps to test it:
 
 1. Connect the development kit's SEGGER J-Link USB port to the PC USB port with a USB cable.
+   If you are using HCI, connect the kit's nRF USB port to the PC USB port instead.
 #. Get the kit's serial port name (for example, :file:`/dev/ttyACM0`).
 #. Run and configure ot-cli as described in :ref:`ug_thread_tools_ot_apps`.
 #. From this point, you can follow the :ref:`ot_cli_sample_testing` instructions in the CLI sample by removing the `ot` prefix for each command.
+#. If you are using HCI, follow the instructions for the :ref`zephyrt:bluetooth-hci-uart-sample` sample in the Zephyr documentation.
```

Comment:
```suggestion
#. If you are using HCI, follow the instructions for the :ref:`zephyrt:bluetooth-hci-uart-sample` sample in the Zephyr documentation.
```
Is there a specific section the user should read in the Zephyr sample documentation, and is this step intended to be done after or instead of the CLI instructions.

Reply:
Last two steps can be done in any order. work of HCI and RCP should be more or less independent.

Reply:
Is there an option to only do the Zephyr instructions when working with HCI?

Reply:
Yes there is.